src/Quizzer.API.DAL/Entities/Question.cs

```diff
@@ -0,0 +1,19 @@
+﻿using AutoMapper;
+
+namespace Quizzer.API.DAL.Entities {
+    public record Question : EntityBase{
+        enum QuestionType { SingleChoice, MultioChoice, OrderChoice}
+
+        QuestionType Type { get; set; }
+        public required string Text { get; set; }
+
+        public Guid QuizId { get; set; }
```

Comment:
Missing navigation property

Reply:
It is not necessery to have the anvigate property everywhere. Do we really need it here?

Reply:
As stated above, or I you have more info than what I provided above?

Reply:
Done